drivers/flash/Kconfig.it8xxx2

```diff
@@ -0,0 +1,17 @@
+# FLASH configuration options
+
+# Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FLASH_ITE_IT8XXX2
+	bool "ITE IT8XXX2 flash driver"
+	depends on SOC_FAMILY_RISCV_ITE
+	select SOC_FLASH_RAMCODE_SECTION
+	help
+	  Enable the ITE it8xxx2 series internal flash driver.
+
+config SOC_FLASH_RAMCODE_SECTION
+	bool
+	help
+	  Enable this config to map the _ram_code_section of flash
```

Comment:
again this needs a lot more detail as from this help I have not idea what this does

Reply:
Done